sphinx/theming.py

```diff
@@ -159,8 +160,13 @@ def _find_themes(theme_path: str) -> dict[str, str]:
                     name = entry[:-4]
                     themes[name] = pathname
                 else:
-                    logger.warning(__('file %r on theme path is not a valid '
-                                      'zipfile or contains no theme'), entry)
+                    logger.warning(
+                        __(
+                            'file %r on theme path is not a valid '
+                            'zipfile or contains no theme'
+                        ),
+                        entry,
+                    )
```

Comment:
Ditto here re: possible variable extraction.  Consistent styling will be great; 2 lines compared to 7 seems a significant difference, though!

Reply:
same as above, the msg won't fit on a single line, so it seems not worth the effort to extract out a `msg` variable